lib/Dialect/TritonGPU/Transforms/WarpSpecialization/OptimizePartitionWarps.cpp

```diff
@@ -127,6 +127,7 @@ static LogicalResult relayoutWarps(ModuleAxisInfoAnalysis &axisInfo,
   pm.addPass(createTritonGPUCoalesce());
   pm.addPass(createTritonGPURemoveLayoutConversions());
   pm.addPass(createTritonGPUOptimizeThreadLocality());
+  pm.addPass(createTritonGPUAccelerateMatmul());
```

Comment:
why do we need to call this one again? The tt.dot should have already been removed.

Reply:
I am running this specifically because the optimization that changes the result layout of TMEMLoadOp to be split along M when it is fed into a reduction is run at the end of this pass... We can split it into a different pass if that makes sense.

Reply:
yeah I think we should separate that out in a different pass. Up to you whether to do it in this PR or a follow up

Reply:
https://github.com/triton-lang/triton/pull/6715